waspc/data/Generator/templates/server/src/auth/providers/oauth/cookies.ts

```diff
@@ -4,14 +4,14 @@ import {
 } from "express";
 import { parseCookies } from "oslo/cookie";
 
-import { type ProviderConfig } from "wasp/auth/providers/types";
+import type { ProviderConfig } from "wasp/auth/providers/types";
 
-import { type StateType } from './state';
+import type { OptionalStateType, RequiredStateType } from './state';
 
 export function setOAuthCookieValue(
   provider: ProviderConfig,
   res: ExpressResponse,
-  stateType: StateType,
+  stateType: RequiredStateType | OptionalStateType,
```

Comment:
Seems like we could make a `type StateType = RequiredStateType | OptionalStateType`

Reply:
I've simplified the logic so this no longer applies.